protein_analysis_tool.py

```diff
@@ -0,0 +1,491 @@
+                if is_amino_acid(letter):
+                    pass
+            result.append(seq)
+        if all(multiple_of_three) and all(test_three_letters):
+            print(
+                "Warning: all your sequences are similar to three-letter ones. Check the letter_format value"
+            )
+        return result
+    elif letter_format == 3:
+        for seq in seqs:
+            seq = seq.lower()
+            seq3 = [seq[i: i + 3] for i in range(0, len(seq), 3)]
+            for triplet in seq3:
+                if is_amino_acid(triplet):
+                    pass
+            seq_transformed = "".join([amino_names_dic.get(seq) for seq in seq3])
+            result.append(seq_transformed)
+        return result
+    else:
+        raise ValueError(
+            "Error unsupported letter_format. Only letter_formats 1 and 3 are supported"
+        )
+
+
+def is_amino_acid(input_amino: str) -> bool:
+    """
+    Checks whether the entered string is an amino acid (either three-letter encoding or one-letter encoded).
+
+    Arguments:
+      - input_amino (str): string corresponding to one amino acid (in three-letter code or one-letter code)
+
+      Return:
+      - bool: True if amino acid is a valid amino acid, otherwise ValueError is amino acid is not correct
+    """
+    if len(input_amino) == 1:
+        letter = input_amino
+        if letter not in amino_short_names_dic.keys():
+            raise ValueError(f"Error {letter} is not an amino acid. Correct your input")
+        return True
+    elif len(input_amino) == 3:
+        triplet = input_amino
+        if triplet not in amino_names_dic.keys():
+            raise ValueError(
+                f"Error {triplet} is not an amino acid. Correct your input"
+            )
+        return True
+    else:
+        raise ValueError(
+            f"Error {input_amino} is incorrect form of amino acid notation. Correct your input"
+        )
+
+
+def brutto_count(seqs: list) -> list:
+    """
+    Calculates the brutto formula of the amino acid sequences.
+
+    Arguments:
+      - seqs (list): list of string with the protein sequences
+
+      Return:
+      - list of dictionaries with counts of each elemet included (elements C,H,N,O,S)"""
+    elements = ["C", "H", "N", "O", "S"]
+    result = []
+    for seq in seqs:
+        brutto_list = [amino_brutto.get(letter) for letter in seq]
+        brutto_pair = list(zip(*brutto_list))
+        brutto = [sum(i) for i in brutto_pair]
+        brutto_dict = dict(zip(elements, brutto))
+        result.append(brutto_dict)
+    return result
+
+
+def is_length_divisible_by_3(seq: str) -> bool:
+    """
+    Checks if the sequence is divisible by three.
+
+    Arguments:
+      - seq (str): string of protein sequence
+
+      Return:
+      - bool: True if sequence is divisible by three, otherwise False"""
+    seq_len = len(seq)
+    if seq_len % 3 == 0:
+        return True
+    else:
+        return False
```

Comment:
Можно убрать else, потому что если код не зайдет в if, то вернет False. А если зайдет в if, вернется True и не будет уже исполнять следующую строку в коде.
```suggestion
    return False
```